docs/docker-compose.yml

```diff
@@ -51,6 +51,13 @@ services:
       SUBLINKS_JWT_SECRET: <JWT-SECRET>
       SUBLINKS_BASE_URL: https://demo.sublinks.org
       SUBLINKS_PICTRS_URL: http://pictrs:8080
+      FEDERATION_QUEUE_HOST: localhost
+      FEDERATION_QUEUE_PORT: 5672
+      FEDERATION_QUEUE_USER: sublinks
+      FEDERATION_QUEUE_PASS: sublinks
+      FEDERATION_QUEUE_NAME: federation # TODO: Need to get correct queue name
+      BACKEND_QUEUE_NAME: backend # TODO: Need to get correct queue name
+      BACKEND_TOPIC_NAME: backend # TODO: Need to get correct topic name
```

Comment:
I don't think these need to be TODO items. Using variables here makes it configurable and having the default be backend/federation is probably fine

Reply:
Sounds good will remove comments